lib/AST/TypeRepr.cpp

```diff
@@ -546,6 +546,12 @@ void SpecifierTypeRepr::printImpl(ASTPrinter &Printer,
   case TypeReprKind::Owned:
     Printer.printKeyword("__owned", Opts, " ");
     break;
+  case TypeReprKind::Isolated:
+    Printer.printKeyword("isolated", Opts, " ");
+    break;
+  case TypeReprKind::CompileTimeConst:
+    Printer.printKeyword("_const", Opts, " ");
+    break;
   default:
```

Comment:
This is why I dislike `default` :)

Reply:
@QuietMisdreavus I am confused how this led to "inout" being printed. Is that the default for function parameters in the SGF? If the default should just be " " it might make sense making the default that 👀

Reply:
It's an issue with builds that have assertions disabled. With assertions enabled, the code (correctly) crashes when seeing `_const` or `isolated`. Without assertions, though, [the `llvm_unreachable` macro](https://github.com/apple/llvm-project/blob/324efdf92b0575ca9003ee5f22e7719f50f68c44/llvm/include/llvm/Support/ErrorHandling.h#L125-L151) turns the branch into UB - which i assume in this case just picks a different branch arbitrarily.

Reply:
Makes sense! should we just remove the default case if no one cares about being exhaustive though?

Reply:
I think having the default there is still useful - this is a problem of test coverage (i don't know what else uses the AST Printer, TBH), not necessarily the switch being incorrect.